apps/src/applab/designElements/dropdown.jsx

```diff
@@ -88,6 +88,16 @@ module.exports = {
     element.style.height = '30px';
     element.style.fontSize = '14px';
     element.style.margin = '0';
+    element.style.color = '#fff';
+    element.style.backgroundColor = '#1abc9c';
```

Comment:
I dont know that we can extra colors from that in JS though. In an ideal world I guess we would have one place that populates both the css and any JS files. Not sure if/how that can be done.